rules/windows/process_creation/proc_creation_win_tool_nircmd.yml

```
@@ -8,7 +8,7 @@ references:
     - https://www.nirsoft.net/utils/nircmd2.html#using
 author: Florian Roth (Nextron Systems), Nasreddine Bencherchali
 date: 2022/01/24
-modified: 2022/11/30
+modified: 2023/02/03
```

```suggestion
modified: 2022/11/30
```
we shouldn't change the modified date for cosmetic changes